xbmc/video/VideoInfoTag.cpp

```diff
@@ -591,6 +598,11 @@ void CVideoInfoTag::ParseNative(const TiXmlElement* movie, bool prioritise)
   {
     m_fRating = m_fRating / max_value * 10; // Normalise the Movie Rating to between 1 and 10
   }
+  const TiXmlElement* urElement = movie->FirstChildElement("userrating");
```

Comment:
Not sure about this, it should be set again by my tag parsing. Not seeing the error here right now.